apis/kueue/v1alpha1/tas_types.go

```diff
@@ -38,6 +38,16 @@ const (
 	// among multiple topology domains.
 	PodSetPreferredTopologyAnnotation = "kueue.x-k8s.io/podset-preferred-topology"
 
+	// PodSetUnconstrainedTopologyAnnotation indicates that a PodSet requires
+	// Topology Aware Scheduling, but it promotes filling up nodes in use over
+	// compact placement. The pods have lower chance of being scheduled on the same
+	// node, but it mitigates resource fragmentation, and can lead to better
+	// node utilization. Recommended for PodSets that don't require heave inter pod
```

Comment:
This seems old, with typos like " require heave inter pod". Please align with KEP.

Reply:
Done